.github/workflows/ubuntu-ci.yml

```diff
@@ -0,0 +1,73 @@
+
+name: Fast DDS Docs Ubuntu CI
+
+on:
+  workflow_dispatch:
+    inputs:
+      os-version:
+        description: 'OS version to run the workflow'
+        required: false
+        default: 'ubuntu-22.04'
+        type: string
+      colcon-args:
+        description: 'Extra arguments for colcon cli'
+        required: false
+        type: string
+      cmake-args:
+        description: 'Extra arguments for cmake cli'
+        required: false
+        type: string
+      ctest-args:
+        description: 'Extra arguments for ctest cli'
+        required: false
+        type: string
+      fastdds-docs-branch:
+        description: 'Branch or tag of Fast DDS Docs repository'
+        type: string
+        required: true
+      fastdds-branch:
+        description: 'Branch or tag of Fast DDS repository'
+        type: string
+        required: true
+      fastdds-python-branch:
+        description: 'Branch or tag of Fast DDS Python repository'
+        required: true
+        type: string
+      run-tests:
+        description: 'Run test suite of Fast DDS python'
+        required: false
+        type: boolean
+        default: true
+      use-ccache:
+        description: 'Use CCache to speed up the build'
+        required: false
+        type: boolean
+        default: false
+
+  pull_request:
+    types:
+      - review_requested
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+      - '!**/CMakeLists.txt'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  ubuntu-ci:
+    uses: ./.github/workflows/reusable-ubuntu-ci.yml
+    with:
+      os-version: ${{ inputs.os-version || 'ubuntu-22.04' }}
+      label: 'ubuntu-ci'
+      colcon-args: ${{ inputs.colcon-args }}
+      cmake-args: ${{ inputs.cmake-args }}
+      ctest-args: ${{ inputs.ctest-args }}
+      fastdds-docs-branch: ${{ inputs.fastdds-docs-branch || github.ref || 'master' }}
+      fastdds-branch: ${{ inputs.fastdds-branch || 'master' }}
+      fastdds-python-branch: ${{ inputs.fastdds-python-branch || 'main' }}
+      run-build: ${{ !(github.event_name == 'pull_request') && true || (!contains(github.event.pull_request.labels.*.name, 'skip-ci') && !contains(github.event.pull_request.labels.*.name, 'conflicts')) && true || false }}
+      run-tests: ${{ ((inputs.run-tests == true) && true) || (github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'no-test')) && true || false }}
```

Comment:
Same comment as for ShapesDemo CI. It might be usefull to move the check on the `conflicts` label before starting the job, so it can appear skipped in the list of the workflows in PRs and not passed